doc/howto/container_gpu_passthrough_with_docker.md

```diff
@@ -0,0 +1,155 @@
+(container-gpu-passthrough-with-docker)=
+# Passing an NVIDIA dGPU / iGPU to a LXD container with a nested Docker workload
```

Comment:
The title should start with "How to ...", and ideally it should be similar to the title in the navigation. And not too long - details should go into the introduction.
So something like "How to pass an NVIDIA GPU to a container" (navigation: "Pass a GPU"/"Pass an INVIDIA GPU") or "How to run GPU workloads on nested docker" (navigation: "Run GPU workloads on docker") or "How to run docker GPU workloads" (navigation: "Run docker GPU workloads") - depends a bit on where you want to put the focus.

Also, are we talking container only? Or do you mean instance?

The clarification that GPU can mean dGPU or iGPU should go into the first paragraph.

Reply:
`How to pass an NVIDIA GPU to a container` seems fine. And yes, this is for container only.

Reply:
fixed